internal/pkg/nats/interfaces/mocks/Connection.go

```diff
@@ -1,9 +1,9 @@
-// Code generated by mockery v2.13.1. DO NOT EDIT.
-// note that nats.go will need to be added to the end of the nats import
+// Code generated by mockery v2.15.0. DO NOT EDIT.
+
 package mocks
 
 import (
-	nats "github.com/nats-io/nats.go"
+	nats "github.com/nats-io/"
```

Comment:
This is causing test failure. 

This is because @alex had to hand modify the generated code. Should restore this to nats "github.com/nats-io/nats.go"

Reply:
done